### PR TITLE
Fix the start script

### PR DESCRIPTION
**Pull request by Miniplop**
## Description

This PR is to solve the regression on the start script where an typescript error occurs : 

Before : 
image

After : 
image

### CODE REVIEW

File: packages/slice-machine/start/index.js [hunk 1]
```diff
@@ -77,11 +77,8 @@ function start({ cwd, port }, callback) {
   });
 
   smServer.stdout.on("data", function (data) {
-    const lns = data.toString().split("=");
+    const lns = data.toString().split("Server running");
```
Comment (Heyrows) [inline packages/slice-machine/start/index.js:80]: The fix that you made by reading the function SMConfig is good, so why did you change this split ?
What does it change ?

Comment (Miniplop) [reply packages/slice-machine/start/index.js:80]: This line of code is here to replace "server running : localhost...." by the green info box that we know

Comment (Miniplop) [reply packages/slice-machine/start/index.js:80]: Before the dev server write "p=9999", now the server write "Server Running: localhost:9999"

Comment (Miniplop) [reply packages/slice-machine/start/index.js:80]: The regression was introduced when we change the Express message in the setup drawer PR

Comment (Heyrows) [reply packages/slice-machine/start/index.js:80]: ah ok i see. I'll trust your screenshots and the validation part to detect if there is an issue but i'm sure there won't be :)

**Review by Heyrows — APPROVED** [review]
